changelog/index.php

```diff
@@ -118,6 +117,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 			$reposToIterate[] = 'recommendations';
 			$reposToIterate[] = 'viewer';
 		}
+		if (!in_array($head, ['stable18', 'stable19']) && !in_array(substr($head, 0, 3), ['v18', 'v19'])) {
```

Comment:
Maybe better white list, so we don't need to think about it all the time?

Reply:
Yep. Would prefer the other way around as well.

Reply:
yeah, better. and done.